seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSource.java

```diff
@@ -186,6 +190,19 @@ public void prepare(Config config) throws PrepareFailException {
                                     this.metadata.getProperties().put(key, value.unwrapped()));
         }
 
+        if (config.hasPath(MESSAGE_FORMAT_ERROR_HANDLE_WAY_OPTION.key())) {
```

Comment:
`Serialization Error Handling Policy ` or `Serialization Failure Handling Policy` maybe better?

Reply:
I mean it is not format error but serialization error